ui/components/component-library/select-button/select-button.scss

```diff
@@ -1,13 +1,55 @@
 .mm-select-button {
-  box-sizing: border-box;
-  // size settings
-  &--type-danger {
-    // placeholder
+  --select-button-height: var(--size, 40px);
+
+  position: relative;
+  cursor: pointer;
+  text-align: start;
+  height: min-content;
+  min-height: var(--select-button-height);
+  overflow: hidden;
+  max-width: 100%;
+
+  &--size-sm {
+    --size: 32px;
+  }
+
+  &--size-md {
+    --size: 40px;
+  }
+
+  &--size-lg {
+    --size: 48px;
+  }
+
+
+  &--block {
+    width: 100%;
+  }
+
+  &:hover {
+    background-color: var(--color-background-hover);
+  }
+
+  &:active {
+    background-color: var(--color-background-pressed);
   }
 
   &:active,
   &:focus {
     border-color: var(--color-primary-default);
     box-shadow: inset 0 0 0 1px var(--color-primary-default);
   }
+
+  &--type-danger {
+    border-color: var(--color-error-default);
```

Comment:
nit: Could we move this to update the borderColor style utility prop to reduce CSS?

Reply:
Roger that